### DISCUSSION TITLE
Bun v1.1.27

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#13792] [Announcements]
To install:
```bash
curl -fsSL https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

Windows:
```bash
powershell -c "irm bun.sh/install.ps1|iex"
```

To upgrade to Bun v1.1.27:
```bash
bun upgrade
```

### Thanks to 17 contributors!

- [@17hz](https://github.com/17hz)
- [@190n](https://github.com/190n)
- [@xales](https://github.com/xales)
- [@nektro](https://github.com/nektro)
- [@RiskyMH](https://github.com/RiskyMH)
- [@fmorency](https://github.com/fmorency)
- [@wpaulino](https://github.com/wpaulino)
- [@mohit-s96](https://github.com/mohit-s96)
- [@Nanome203](https://github.com/Nanome203)
- [@jakebailey](https://github.com/jakebailey)
- [@sacsbrainz](https://github.com/sacsbrainz)
- [@jakeboone02](https://github.com/jakeboone02)
- [@cirospaciari](https://github.com/cirospaciari)
- [@dylan-conway](https://github.com/dylan-conway)
- [@Marukome0743](https://github.com/Marukome0743)
- [@Jarred-Sumner](https://github.com/Jarred-Sumner)
- [@SunsetTechuila](https://github.com/SunsetTechuila)

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.1.27'>Bun v1.1.27</a>.</em>

**Comment by 7heMech** [comment]
So currently I have a handler to serve static files in bun, and it adds headers like length and type + checks for which compression methods client supports and has compressed the static files at build-step, so it serves them compressed if client supports it.
Does this new static feature handle compression?

**Comment by truongan07** [comment]
@Jarred-Sumner Hi friend, high priority implementation PostgresSQL and HTTP/2, please.

**Reply by 7heMech** [reply]
You're welcome to contribute if you want something impl faster.
